{
  try {
    methodThatThrowsAnError();
  }
 catch (  Error e) {
    throw new Exception("Throws an exception",e);
  }
}
